Website/DesktopModules/Hotcakes/Core/Admin/Parts/Shipping/UPSFreight/App_LocalResources/Edit.ascx.resx

```diff
@@ -0,0 +1,408 @@
+  </data>
+  <data name="AllAvailable.Text" xml:space="preserve">
+    <value>Try all available UPS Services</value>
+  </data>
+  <data name="AlreadyRegistered.Text" xml:space="preserve">
+    <value>Already Registered with UPS (click to register again)</value>
+  </data>
+  <data name="Amount.Text" xml:space="preserve">
+    <value>Amount</value>
+  </data>
+  <data name="AvailableServices.Text" xml:space="preserve">
+    <value>Available Services</value>
+  </data>
+  <data name="btnCancel.Text" xml:space="preserve">
+    <value>Cancel</value>
+  </data>
+  <data name="btnSave.Text" xml:space="preserve">
+    <value>Save Changes</value>
+  </data>
+  <data name="CustomerCounter.Text" xml:space="preserve">
+    <value>Customer Counter</value>
+  </data>
+  <data name="CustomerSupplied.Text" xml:space="preserve">
+    <value>Customer Supplied</value>
+  </data>
+  <data name="cvAdjustmentTextBox.ErrorMessage" xml:space="preserve">
+    <value>Adjustment price must be numeric (decimals are okay).</value>
+  </data>
+  <data name="DailyPickup.Text" xml:space="preserve">
+    <value>Daily Pick-Up</value>
+  </data>
+  <data name="DefaultPackaging.Text" xml:space="preserve">
+    <value>Default Packaging</value>
+  </data>
+  <data name="DefaultService.Text" xml:space="preserve">
+    <value>Default Service</value>
+  </data>
+  <data name="DiagnosticsMode.Text" xml:space="preserve">
+    <value>Diagnostics Mode</value>
+  </data>
+  <data name="DontUseDimensions.Text" xml:space="preserve">
+    <value>Don't Use Package Dimensions (only weight)</value>
+  </data>
+  <data name="EditShippingMethod.Text" xml:space="preserve">
+    <value>Edit Shipping Method: UPS</value>
+  </data>
+  <data name="ForceResidentialAddresses.Text" xml:space="preserve">
+    <value>Force Residential Addresses</value>
+  </data>
+  <data name="HighlightColor.Text" xml:space="preserve">
+    <value>Highlight Color</value>
+  </data>
+  <data name="Name.Text" xml:space="preserve">
+    <value>Name</value>
+  </data>
+  <data name="OnlySelected.Text" xml:space="preserve">
+    <value>Try only the UPS Services I select</value>
+  </data>
+  <data name="Percentage.Text" xml:space="preserve">
+    <value>Percentage</value>
+  </data>
+  <data name="PickupType.Text" xml:space="preserve">
+    <value>Pick-Up Type</value>
+  </data>
+  <data name="RegisterWithUPS.Text" xml:space="preserve">
+    <value>Register with UPS to use their online tools</value>
+  </data>
+  <data name="RegistrationStatus.Text" xml:space="preserve">
+    <value>Registration Status</value>
+  </data>
+  <data name="rfvAdjustmentTextBox.ErrorMessage" xml:space="preserve">
+    <value>Adjustment price is required.</value>
+  </data>
+  <data name="ServicesToUse.Text" xml:space="preserve">
+    <value>Service(s) to Use</value>
+  </data>
+  <data name="ShippingZone.Text" xml:space="preserve">
+    <value>Shipping Zone</value>
+  </data>
+  <data name="UPS10KgBox.Text" xml:space="preserve">
```

Comment:
Are these standard UPS package and shipping keys/values necessary for the freight shipping method?

Reply:
i have re-verify all keys and removed which are not used for freight shipping method.